include/exadg/operators/interior_penalty_parameter.h

```diff
@@ -89,16 +93,26 @@ calculate_penalty_parameter(
 }
 
 /*
- *  This function returns the penalty factor of the interior penalty method
- *  for quadrilateral/hexahedral elements for a given polynomial degree of
- *  the shape functions and a specified penalty factor (scaling factor).
+ *  This function returns the penalty factor of the interior penalty method for
+ *  quadrilateral/hexahedral or for triangular/tetrahedral elements for a given
+ *  polynomial degree of the shape functions, a specified penalty factor
+ *  (scaling factor) and the dimension of the problem.
  */
 
-template<typename Number>
+template<int dim, typename Number>
 Number
-get_penalty_factor(unsigned int const degree, Number const factor = 1.0)
+get_penalty_factor(unsigned int const degree, ElementType element_type, Number const factor = 1.0)
 {
-  return factor * (degree + 1.0) * (degree + 1.0);
+  if(element_type == ElementType::Simplex)
+    // use penalty factor for simplex elements according to K. Shahbazi, An explicit expression for
+    // the penalty parameter of the interior penalty method, Journal of Computational Physics 205,
+    // 401–407, 2005.
+    return factor * (degree + 1.0) * (degree + dim) / dim;
+  else
```

Comment:
should be `else if(element_type == Hypercube)`. An additional else branch with `AssertThrow(false,...)` is required below.